app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    before_action :authenticate_user!
```

Comment:
Hey this line probably needs to be in the controller...?

/sorry siu sing messed up dyana desu